jax/core.py

```diff
@@ -652,7 +652,7 @@ def pp_eqn(eqn):
           >> pp(' [ {} ; {} ]'.format(pp_vars(const_vars),
                                       pp_vars(bound_vars))))
   return (pp('{} = '.format(lhs)) >>
-          pp(eqn.primitive.name) >> pp_kv_pairs(eqn.params.items())
+          pp(eqn.primitive.name) >> pp_kv_pairs(eqn.params)
```

Comment:
Maybe rename `pp_kv_pairs` to `pp_dict` or something, since the name no longer reflects the type?

Also because tuple sorting is lexical, I think you could alternatively keep the previous function `pp_kv_pairs` unmodified and have written here `pp_kv_pairs(sorted(eqn.params.items()))`.

Reply:
I will do the `sorted(...items()` but keep the pp_kv_pairs, because then those are pairs again